package.json

```diff
@@ -20,6 +20,13 @@
     "webpack-shell-plugin": "^0.5.0"
   },
   "scripts": {
-    "build": "./node_modules/webpack/bin/webpack.js"
+    "build": "webpack"
```

Comment:
npm scripts add the "./node_modules/.bin/" to the `$PATH`, so we can just use `webpack` here.